supernode/daemon/mgr/ha/etcd_tool.go

```diff
@@ -0,0 +1,146 @@
+package ha
+
+import (
+	"context"
```

Comment:
please follow the rule https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#rule003---import-packages to import packages. Thanks

Reply:
done